### PR TITLE
stubbing tutorial section of docs

### PR DESCRIPTION
**Pull request by brianehlert**
stubbing a new tutorial section to be populated by contributors which are being sorted.

### Proposed changes
This is the establishment of a new tutorials section of the documentation.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [ x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ x] I have updated necessary documentation
- [ ] I have rebased my branch onto master
- [ ] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by tomasohaodha — APPROVED** [review]

File: docs/content/troubleshooting/_index.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
 title: Troubleshooting
+description:
```
Comment (jputrino) [inline docs/content/troubleshooting/_index.md:3]: Description should be provided for SEO purposes

File: docs/content/troubleshooting/_index.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
 weight: 2000
+menu:
```
Comment (jputrino) [inline docs/content/troubleshooting/_index.md:5]: I don't think this menu metadata actually is being used at present, but it doesn't hurt anything to be here.

File: docs/content/troubleshooting/_index.md (hunk 1 above)
Comment (jputrino) [reply docs/content/troubleshooting/_index.md:3]: ... well, SEO and to help users figure out what's in that section :)

**Review by jputrino — APPROVED** [review]

File: docs/content/troubleshooting/_index.md (hunk 2 above)
Comment (jputrino) [reply docs/content/troubleshooting/_index.md:5]: Also, if you want these to be present for later use but not actually build (and therefore publish), add the following to the frontmatter:

```yaml
_build:
  render: never
  list: never
```

**Review by jputrino — APPROVED** [review]

**Pull request merged**